clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -0,0 +1,267 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.clients.admin;
+
+import org.apache.kafka.common.Node;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * The interface for the {@link AdminClient}
+ * @see KafkaAdminClient
+ */
+public abstract class AdminClient implements AutoCloseable {
+    /**
+     * Create a new AdminClient with the given configuration.
+     *
+     * @param conf          The configuration.
+     * @return              The new KafkaAdminClient.
+     */
+    public static AdminClient create(Map<String, Object> conf) {
+        return KafkaAdminClient.create(new AdminClientConfig(conf));
+    }
+
+    /**
+     * Close the AdminClient and release all associated resources.
+     */
+    public abstract void close();
+
+    /**
+     * Create a new topic with the default options.
+     *
+     * @param newTopic          The new topic to create.
+     * @return                  The CreateTopicsResults.
+     */
+    public CreateTopicResults createTopic(NewTopic newTopic) {
```

Comment:
I suppose it is easier to add new methods than to remove them.  I will remove the singular methods for now, and add the \@Unstable annotation to the API classes.